fieldservice/models/service_request.py

```diff
@@ -0,0 +1,183 @@
+# Copyright 2017 Ursa Information Systems <http://www.ursainfosystems.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+# from odoo import api, _
+
+
+class ServiceLocation(models.Model):
+    _name = 'service.location'
+    _description = 'Location of the service'
+
+    # customer_id = fields.Many2one('res.partner', string='Customer')
+    name = fields.Char(string='Name')
+    location_type = fields.Char(string='Location Type', size=35)
+    # building = fields.Char(string='Building', size=35)
+    # floor = fields.Char(string='Floor', size=35)
+    # longitude = fields.Float(string='Longitude')
+    # latitude = fields.Float(string='Latitude')
+    description = fields.Char(string='Description')
+    # branch = fields.Char(string='Branch', size=35)
+    # territory = fields.Char(string='Territory', size=35)
+    # timezone = fields.Char(string='TimeZone', size=35)
+
+
+class ServiceRequest(models.Model):
+    _name = 'service.request'
+    _description = 'Details of the Service Request'
```

Comment:
Is this for Work Orders inside a Service Order?
Should it have a `order_id` field for the parent Order?